testing_and_setup/compass/ocean/global_ocean/config_files/config_mesh_metrics.xml

```diff
@@ -6,15 +6,14 @@
 
 	<add_link source="../culled_mesh/culled_mesh.nc" dest="mesh.nc"/>
 	<add_link source_path="bathymetry_database" source="SRTM15_plus_earth_relief_15s.nc" dest="earth_relief_15s.nc"/>
-	<add_link source_path="mpas_model" source="testing_and_setup/compass/ocean/jigsaw_to_MPAS/inject_bathymetry.py" dest="inject_bathymetry.py"/>
 
 	<run_script name="run.py">
 		<step executable="MpasMeshConverter.x">
 			<argument flag="mesh.nc"></argument>
 			<argument flag="mesh_with_metrics.nc"></argument>
 		</step>
 
-		<step executable="./inject_bathymetry.py">
```

Comment:
Is this correct?  This appears to be a local script and sometimes these were not local scripts.  May need to be careful here.

Reply:
It is just different syntax for the same call.  In some cases, a local link was being made to `inject_bathymetry.py` itself (as here) and in other cases the link is to the `jigsaw_to_MPAS` module, so you need to call the same script with `python -m jigsaw_to_MPAS.inject_bathymetry` instead of `./inject_bathymetry.py`.  It's always the same script, which will now be an entry point instead.